src/cider/nrepl/middleware/pprint.clj

```diff
@@ -120,7 +120,7 @@
   false, or string representations of the above)."
   [handler]
   (fn [{:keys [op pprint] :as msg}]
-    (handler (if (and pprint (= op "eval"))
+    (handler (if (and pprint (#{"eval" "load-file"} op))
```

Comment:
I'm not sure I understand what this is supposed to do. The op is one of the two?